conf/mirrors.yaml

```diff
@@ -108,15 +108,15 @@
   country: za
   flag: za
   name: Stellenbosch University, South Africa
-  url: ftp://ftp.sun.ac.za/pub/mirrors/www.sagemath.org/
-- active: true
+  url: http://ftp.sun.ac.za/ftp/pub/mirrors/www.sagemath.org/
+- active: false
```

Comment:
why is it deactivated? http://ftp.sun.ac.za/ftp/pub/mirrors/www.sagemath.org/ loads fine for me.

Reply:
oh, next one, I see ^^

Reply:
Yes, it's a bit confusing in the diff view, but the structure is

```
- active: xx  # true/false --- whether mirror is active
  cat: xx     # a/af/aus/e/na/sa for Asia/Africa/Oceania/Europe/NAm/SAm
  country: xx           # country code
  flag: xx              # country code for the flag
  name: xx, yy, zz      # name (eg Uni Something), city, country
  priority: 1000        # priority: 1 master, 10 secondary master, ...
  url: http://example.com/mirror/sage/  # url --- must end with /
```

and the diff view often has the name and url of one mirror
combined with the active status of the next mirror.